### PR TITLE
[14.0][ADD] partner_phone_secondary

### PR DESCRIPTION
**Pull request by ivantodorovich**
Adds a secondary phone field in `res.partner`

### CODE REVIEW

**Comment by newtratip** [comment]
@ivantodorovich, Please add test script to cover codecov.

**Review by HaraldPanten — APPROVED** [review]

**Comment by ivantodorovich** [comment]
> @ivantodorovich, Please add test script to cover codecov.

done

**Review by newtratip — APPROVED** [review]
Test with only runbot.
Selection_579

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Comment by emagdalenaC2i** [comment]
@OCA/community-maintainers Please, could you merge this PR?

File: partner_phone_secondary/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 - Iván Todorovich
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```
Comment (NL66278) [inline partner_phone_secondary/models/res_partner.py]: Make it https:

Personally I would like to have this as a header in all files, certainly also in the manifest file.

Comment (ivantodorovich) [reply partner_phone_secondary/models/res_partner.py]: done

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

**Pull request merged**